lib/main.ts

```diff
@@ -112,7 +118,7 @@ export function idTokenCustomClaims<T extends object>(): Omit<
 > {
   if (!kinde.idToken) {
     throw new Error(
-      "IdToken binding not available, please add to workflow settings to enable",
+      "IdToken binding not available, please add to workflow/page settings to enable",
```

Comment:
@DanielRivers I don't think we need `page` here as I don't think we'd expose an ID token within the page. Same for access token